index.d.ts

```diff
@@ -13,7 +13,7 @@ export interface QRCodeProps {
   /* the color of the background */
   backgroundColor?: string;
   /* an image source object. example {uri: 'base64string'} or {require('pathToImage')} */
-  logo?: ImageSourcePropType;
+  logo?: ImageSourcePropType | string;
```

Comment:
Why adding example app adds this change?

Reply:
I will remove it 👍 you're right its confusing